icons/f-secure.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" aria-labelledby="simpleicons-fsecure-icon" role="img" viewBox="0 0 24 24"><title id="simpleicons-fsecure-icon">F-Secure icon</title><path d="M25.49 2.24A37.85 37.85 0 0 0 1.8 1.6C.23 2.08-.2 3.42.08 4.56c.42 1.73 1.04 3.7 1.78 5.5.01.04.04.02.03-.02-.09-.84.54-1.9 1.8-2.3A29.06 29.06 0 0 1 22.2 8c.92.32 1.95-.18 2.24-1.13a38.82 38.82 0 0 0 1.12-4.49c.01-.07-.05-.12-.08-.13zm0 0"/><path d="M7.09 7.57c-.92.2-2.2.5-3.13.82-1.9.66-1.82 3-.8 3.74.07-.43.52-1.02 1.04-1.22a21.7 21.7 0 0 1 5.98-1.39A10.95 10.95 0 0 1 7.1 7.57zm0 0"/><path d="M22.2 12.25a22.31 22.31 0 0 0-17.54-.7c-.89.34-1.46 1.45-.89 2.5a41.18 41.18 0 0 0 3.35 5.18c-.18-.62.14-1.8 1.35-2.17 3.35-1.04 6.88-.6 9.23.35.66.27 1.59.11 2.12-.69a41.77 41.77 0 0 0 2.43-4.32c.04-.06 0-.12-.05-.15zm0 0"/><path d="M16.75 20.61c-.6-.17-2.09-1.35-2.66-1.93-.37-.37-.9-.9-1.47-1.52-1.28 0-2.53.13-3.86.55-1.22.4-1.34 1.85-.8 2.6.92 1.14 1.55 1.77 2.58 2.84a2.9 2.9 0 0 0 4.07.03c.8-.82 1.29-1.36 2.2-2.4.04-.05.03-.14-.06-.17zm0 0"/></svg>
```

Comment:
It looks like something went wrong here 🤔 

![image](https://user-images.githubusercontent.com/3742559/47339367-ade0bc00-d6a3-11e8-8240-9961f46763c4.png)